test/e2e/gomega_util.go

```diff
@@ -59,3 +105,178 @@ func eventuallyFindPVs(t *testing.T, f *framework.Framework, storageClassName st
 	return matchedPVs
 
 }
+
+// waits for PVs of the same name to become available
+func eventuallyFindAvailablePVs(t *testing.T, f *framework.Framework, storageClassName string, expectedPVs []corev1.PersistentVolume) []corev1.PersistentVolume {
+	matcher := gomega.NewWithT(t)
+	var newPVs []corev1.PersistentVolume
+	matcher.Eventually(func() bool {
+		newPVs = eventuallyFindPVs(t, f, storageClassName, len(expectedPVs))
+		for _, pv := range expectedPVs {
+			pvFound := false
+			for _, newPV := range newPVs {
+				if pv.Name == newPV.Name {
+					if newPV.Status.Phase == corev1.VolumeAvailable {
+						pvFound = true
+					} else {
+						t.Logf("PV is in phase %q, waiting for it to be in phase %q", newPV.Status.Phase, corev1.VolumeAvailable)
+					}
+					break
+				}
+			}
+			// expect to find each pv
+			if !pvFound {
+				return false
+			}
+		}
+		return true
+	}, time.Minute*8, time.Second*25).Should(gomega.BeTrue(), "waiting for PVs to become available again")
+	return newPVs
+}
+
+func consumePV(t *testing.T, ctx *framework.Context, pv corev1.PersistentVolume) (*corev1.PersistentVolumeClaim, *batchv1.Job, *corev1.Pod) {
+	t.Logf("consuming PV: %q", pv.Name)
+	matcher := gomega.NewWithT(t)
+	f := framework.Global
+	name := fmt.Sprintf("%s-consumer", pv.ObjectMeta.Name)
+	pvc := &corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: f.WatchNamespace,
```

Comment:
Its changed to `f.OperatorNamespace` in the latest test-frameork.

Reply:
Ah thanks! Was just looking for the right one. When I checked the framework code, I thought I saw WatchNamespace getting populated, but looks like it's blank